packages/screens/AllProjectAdministrationDash/AllProjectAdministrationDashScreen.tsx

```diff
@@ -0,0 +1,385 @@
+      }}
+    >
+      <BrandText
+        style={[
+          { flex: TABLE_ROWS.rank.flex },
+          isMobile ? fontSemibold11 : fontSemibold13,
+        ]}
+      >
+        {rowData.rank}
+      </BrandText>
+      <View
+        style={{
+          flex: TABLE_ROWS.collectionNameData.flex,
+          flexDirection: "row",
+          flexWrap: "nowrap",
+          alignItems: "center",
+          paddingRight: layout.spacing_x1,
+        }}
+      >
+        <RoundedGradientImage
+          size="XS"
+          sourceURI={avaPNG}
+          style={{ marginRight: isMobile ? 8 : 16 }}
+        />
+        <BrandText
+          style={[
+            isMobile ? fontSemibold11 : fontSemibold13,
+            { marginRight: 8 },
+          ]}
+        >
+          {rowData.collectionNameData}
+        </BrandText>
+        <ShieldCheckIcon color={primaryColor} width={20} height={20} />
+      </View>
+      <InnerCell style={{ flex: TABLE_ROWS.floor.flex }}>
+        {rowData.floor}
+      </InnerCell>
+      {!isMobile && (
+        <>
+          <InnerCell style={{ flex: TABLE_ROWS.totalVol.flex }}>
+            {rowData.totalVol}
+          </InnerCell>
+          <InnerCell style={{ flex: TABLE_ROWS.vol.flex }}>
+            {rowData.vol}
+          </InnerCell>
+          <PercentageVolume
+            data={rowData.volPerctage}
+            style={{ flex: TABLE_ROWS.volPerctage.flex }}
+          />
+        </>
+      )}
+    </View>
+  );
+};
+
+const InnerCell: React.FC<{
+  style?: StyleProp<ViewStyle>;
+  textStyle?: StyleProp<TextStyle>;
+  children: ReactNode;
+}> = ({ children, style, textStyle }) => {
+  const isMobile = useIsMobile();
+  return (
+    <View
+      style={[
+        {
+          paddingRight: layout.spacing_x1,
+          flexDirection: "row",
+          flexWrap: "nowrap",
+          alignItems: "center",
+        },
+        style,
+      ]}
+    >
+      <SVG source={cryptoLogoSVG} />
+      <BrandText
+        style={[
+          isMobile ? fontSemibold11 : fontSemibold13,
+          { marginLeft: 8 },
+          textStyle,
+        ]}
+        numberOfLines={1}
+      >
+        {children}
+      </BrandText>
+    </View>
+  );
+};
+
+const PercentageVolume: React.FC<{
+  data: any;
+  style?: StyleProp<ViewStyle>;
+}> = ({ data, style }) => {
+  const isMobile = useIsMobile();
+  return (
+    <View
+      style={[
+        {
+          paddingRight: layout.spacing_x1,
+          flexDirection: "row",
+          flexWrap: "nowrap",
+          alignItems: "center",
+        },
+        style,
+      ]}
+    >
+      {data.includes("+") ? (
+        <View
+          style={{
+            backgroundColor: "#C8FFAE33",
+            padding: 5,
+            borderRadius: 100,
+          }}
+        >
+          <ArrowUpIcon color={successColor} width={16} height={16} />
+        </View>
+      ) : (
+        <View
+          style={{
+            backgroundColor: "#FFAEAE33",
```

Comment:
does this color exist already in `colors` file ? if it's specific color related to this feature then it's fine to use hardcode color here

Reply:
Fixed, PR updated